### PR TITLE
use proper file extension when copying config files

### LINKED ISSUE #48: Yaml tests fail with:

### PR DESCRIPTION
**Pull request by nkubala**
fixes #48 

@dlorenc

### CODE REVIEW

File: structure_tests/ext_run.sh [hunk 1]
```diff
@@ -74,8 +74,20 @@ while test $# -gt 0; do
...
+						;;
+					*yaml)
+						EXTENSION=".yaml"
+						;;
+					*)
+						echo "please provide valid JSON or YAML file: $1"
+						exit 1
+						;;
+				esac
+				cp "$1" "$CONFIG_DIR"/cfg_$CONFIG_COUNTER$EXTENSION
```
Comment (dlorenc) [inline structure_tests/ext_run.sh]: Could we simplify this by just copying the entire file name? Then you wouldn't need to do the counters either.

I think this is possible now that we just cleanup the entire tmpdir.

Comment (nkubala) [inline structure_tests/ext_run.sh]: sure, thats much cleaner. done.

**Comment by nkubala** [comment]
also, added a quick line to remove the intermediate structure test container if it exists on the host machine (this happens when the script unexpectedly fails)

File: structure_tests/ext_run.sh [hunk 2]
```diff
@@ -74,9 +73,19 @@ while test $# -gt 0; do
...
+				case "$1" in
+					*json)
+						;;
+					*yaml)
+						;;
+					*)
+						echo "please provide valid JSON or YAML file: $1"
+						exit 1
+						;;
+				esac
```
Comment (dlorenc) [inline structure_tests/ext_run.sh]: Just drop the case here? I think we already validate in the .go file.

Comment (nkubala) [inline structure_tests/ext_run.sh]: ah, good call.

**Pull request merged**